makefiles/suit.v4.inc.mk

```diff
@@ -62,6 +63,9 @@ suit/notify: | $(filter suit/publish, $(MAKECMDGOALS))
 		--payload "$(SUIT_COAP_ROOT)/$$(basename $(SUIT_MANIFEST_SIGNED_LATEST))" && \
 		echo "Triggered $(SUIT_CLIENT) to update."
 
+suit/delkeys:
+	rm -f *key*
```

Comment:
Looks a bit dirty. I think this should be improved, including the gen_key.py script.